components/clp-package-utils/clp_package_utils/scripts/start_clp.py

```
@@ -1111,6 +1115,89 @@
+    container_start_cmd.append(clp_config.execution_container)
+
+    # fmt: off
+    retention_cleaner_cmd = [
+        "python3", "-u",
+        "-m", "job_orchestration.retention.retention_cleaner",
+        "--config", str(container_clp_config.logs_directory / container_config_filename),
+    ]
+    cmd = container_start_cmd + retention_cleaner_cmd
+    subprocess.run(cmd, stdout=subprocess.DEVNULL, check=True)
+
+    logger.info(f"Started {component_name}.")
```

Pylint (3.3.7)

[warning] Use lazy % formatting in logging functions

(W1203)